share/completions/open.fish

```diff
@@ -1,14 +0,0 @@
-if test (uname) = 'Darwin' # OS X
```

Comment:
These are the tab completions for the native Darwin `open` tool - these should probably be kept, right?